### PR TITLE
fixing #8362

### PR DESCRIPTION
**Pull request by hasuniea**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx [hunk 1]
```diff
@@ -124,39 +124,52 @@ function TryOutController(props) {
         let keys;
         let selectedKeyTypes = 'PRODUCTION';
         let accessToken;
-        const promiseSubscriptions = restApi.getSubscriptions(apiID);
-        promiseSubscriptions.then((subscriptionsResponse) => {
-            if (subscriptionsResponse !== null) {
-                subscriptionsList = subscriptionsResponse.obj.list.filter((item) => item.status === 'UNBLOCKED'
-                || item.status === 'PROD_ONLY_BLOCKED');
+        if (!api.lifeCycleStatus && api.lifeCycleStatus.toLowerCase() === 'prototyped') {
```
Comment (kavishkafernando) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx:127]: shouldn't the subscription call happen if api.lifeCycleStatus.toLowerCase() !== 'prototyped') ?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx [hunk 2]
```diff
@@ -169,77 +182,68 @@ function TryOutController(props) {
...
-            })
-            .catch((error) => {
-                console.error(error);
-                const { status } = error;
-                if (status === 404) {
-                    setNotFound(true);
-                }
-                setIsUpdating(false);
-            });
+        if (!api.lifeCycleStatus && api.lifeCycleStatus.toLowerCase() === 'prototyped') {
```
Comment (kavishkafernando) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx:218]: shouldn't the subscription call happen if api.lifeCycleStatus.toLowerCase() !== 'prototyped') ?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx (hunk 1 above)
Comment (hasuniea) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx:126]: fixed issues

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx (hunk 2 above)
Comment (hasuniea) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx]: fixed issue.

**Pull request merged**